src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.dto.CinemaHallRequestDto;
+import mate.academy.spring.model.dto.CinemaHallResponseDto;
+import mate.academy.spring.security.mapper.CinemaHallDtoMapper;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema_halls")
```

Comment:
```suggestion
@RequestMapping("/cinema-halls")
```
https://hostiq.ua/blog/ukr/what-is-url/